### PR TITLE
[P1] Radar Card Common Component

### PR DESCRIPTION
**Pull request by kalachou**
Now, we need to show more then one card.
image

### CODE REVIEW

**Review by vladimir-sviridenko — APPROVED** [review]

File: src-fe/src/app/libs/radars-general-view/radars-general-view.component.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+		<div class="radars-general-view__content-wrapper">
+			<app-radar-card
+				class="radars-general-view__radar-card"
+				*ngFor="let radar of this.radars; let index=index"
+				(click)="navigateToRadar(radar.id)"
+				[title]="radar.name"
+				[lastUpdatedDate]="radar.lastUpdatedDate"
+				[theme]="theme$ | async">
+				<app-general-radar-chart [radar]="radar" [radarDataItems]="radarDataItems[index]" *ngIf="radar && radarDataItems[index]"></app-general-radar-chart>
```
Comment (clain-atomi) [inline src-fe/src/app/libs/radars-general-view/radars-general-view.component.html:20]: lets move ngIf expression to method inside component

File: src-fe/src/app/libs/radars-general-view/store/radars-general-view/radars-general-view.effects.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import { Radar, RadarDto } from '../../../radar-view/model/radar';
+import { RadarConverterService } from '../../../radar-view/service/radar-converter.service';
+
+@Injectable()
+export class RadarsGeneralViewEffects {
+	@Effect()
+	public loadRadars$: Observable<Action> = this.actions$.pipe(
+		ofType(RadarsGeneralViewActionTypes.LoadAllLatestRadars),
+		switchMap((action: LoadAllLatestRadars) => {
+			return this.radarsGeneralViewRepositoryService.loadAllLatestRadars().pipe(
```
Comment (clain-atomi) [inline src-fe/src/app/libs/radars-general-view/store/radars-general-view/radars-general-view.effects.ts:23]: too long name, lets use at least `this.radarsGeneralViewRepository`

File: src-fe/src/app/libs/radars-general-view/store/radars-general-view/radars-general-view.reducer.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+export const initialRadarsGeneralViewState: RadarsGeneralViewState = {
+	radars: {
+		value: null,
+		status: EntityStatus.Init,
+	},
+};
+
+export function radarsGeneralViewReducer(
+	state: RadarsGeneralViewState = initialRadarsGeneralViewState,
+	action: RadarsGeneralViewActions
```
Comment (clain-atomi) [inline src-fe/src/app/libs/radars-general-view/store/radars-general-view/radars-general-view.reducer.ts:14]: lets try to simplify it to one line

**Review by clain-atomi — CHANGES_REQUESTED** [review]

**Comment by clain-atomi** [comment]
can be merged after fix

File: src-fe/src/app/libs/radars-general-view/store/radars-general-view/radars-general-view.reducer.ts (hunk 3 above)
Comment (kalachou) [reply src-fe/src/app/libs/radars-general-view/store/radars-general-view/radars-general-view.reducer.ts:14]: line-max-length is exceeded in the case

**Pull request merged**